.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,196 @@
+name: ci
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+    services:
+      postgres:
+        image: postgres:12
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies for Ubuntu
+      run: |
+        sudo apt-get install neofetch
+        sudo apt-get install -q -y libmecab-dev swig mecab
+        # In Ubuntu 18.04, unidic-mecab is too old, so we install it manually. Note
+        # that this automatically sets the dictionary to unidic.
```

Comment:
This is actually not true.
The reason is that the unidic-mecab that will be installed on 18.04 by default does not include a dictionary as described one line below.

Reply:
Thanks, I've updated the comment.